src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends ColorSupplier {
```

Comment:
The Lottery must not extend the ColorSupplier,
Create a ColorSupplier object just like you did with Random
``` suggestion
public class Lottery {
```